std_navbar-prerelease.ddoc

```diff
@@ -121,6 +121,16 @@ $(DIVID cssmenu, $(UL
 		  $(MODULE3 core, sync, semaphore)
 		)
 	  )
+	$(MENU_W_SUBMENU $(TT std.experimental))
+	  $(ITEMIZE
+		$(MODULE3 std, experimental, logger, package),
```

Comment:
No, the group caption should say "logger", not "package". Look e.g. at how std.container is presented in the navigation.